### DISCUSSION TITLE
Why intercept() some times doesn't catch the request

### DISCUSSION THREAD

**Discussion by xwmyth8023** [cypress-io/cypress#14718] [Questions and Help]
Sometimes, `intercept()` can catch the request, but sometimes can't, anyone can give a hand?
This is a successful case:
![image](https://user-images.githubusercontent.com/29378004/105582793-e879b780-5dca-11eb-89a8-a3502aac864d.png)
But when re-test it, it gets failed
![image](https://user-images.githubusercontent.com/29378004/105584014-45756d80-5dcb-11eb-9341-acc089060397.png)

Example code:
```
it("test", () => {
    cy.intercept("GET", "https://api.segment.io/v1/pixel/track", (req) => {
      expect(req.url).to.include('/v1/pixel/track')
      req.reply((res) => {
        expect(res.url).to.include('/v1/pixel/track')
      })
      if (req.url.includes('/v1/pixel/track')) {
        req.alias = 'gqlMutation'
      }
    })
    cy.get("#baby-name-home-nav").click()
    
    cy.wait("@gqlMutation").its('response').then(res => {
     console.log(res)
    })
 })
```

**Comment by dillibabuj** [comment]
@xwmyth8023  Are you sure the request is being made to `'v1/pixel/track' ` every time you visit manually ? and it's only failing when you are trying it with cypress ? In the screenshots I don't see request to `'v1/pixel/track' ` in the command log - so just want to double check before I suggest something. Please check that and let me know. If that's not the case provide any sample repo I can use to help you better.

**Reply by xwmyth8023** [reply]
I'm sure the request is being made to  `v1/pixel/track` every time visit manually. But most of the time it's failing when trying it in cypress.The example code you can use to check, Just add this code `cy.visit('https://www.thebump.com/baby-names/amp')` before intercept action. Thanks!
